### PR TITLE
Feat/165 update summarycostscsvfombuilder

### PR DESCRIPTION
**Pull request by ArdtK**
## Issue addressed
Solves <issue number>

## Code of conduct
- [x] I HAVE NOT added sensitive or compromised (test) data to the repository.
- [x] I HAVE NOT added vulnerabilities to the repository.
- [ ] I HAVE discussed my solution with (other) members of the KOSWAT team.

## What has been done?
Explain how you addressed the resolution of the related issue, what choices you made and why.

### Checklist
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [x] Updated documentation if needed.

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

File: koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py [hunk 1]
```diff
@@ -108,6 +117,65 @@ def _get_summary_reinforcement_type_column_order(
             for _report in self.koswat_summary.locations_profile_report_list
         ]
 
+    def _get_total_cost(
```
Comment (Carsopre) [inline koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py:120]:
```suggestion
    def _get_total_infrastructure_cost(
```

**Review by Carsopre — COMMENTED** [review]
Just a small remark on regards of method naming.

File: koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py (hunk 1 above)
Comment (ArdtK) [reply koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py:120]: Actually it is the total cost. For infrastructure cost a different method is created.
The total cost is the sum of the measure cost and infrastructure cost.

Comment (ArdtK) [reply koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py:120]: I would propose to keep it like this. The output format is similar to `_get_infrastructure_cost` and `_get_cost_per_selected_measure`.

Comment (Carsopre) [reply koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py:120]: I still want to reduce the ambiguity, please change it as discussed via teams to a more descriptive option.

**Review by Carsopre — APPROVED** [review]

**Pull request merged**